addons/website_sale_product_configurator/static/tests/tours/website_sale_stock_message_after_close.js

```diff
@@ -0,0 +1,61 @@
+odoo.define('website_sale.website_sale_stock_message_after_close', function (require) {
```

Comment:
The two tours must be defined in `website_sale_stock_product_configurator`, they fail when `website_sale_stock` is not installed.